### PR TITLE
Upgrade npm and fix npm warnings

### LINKED ISSUE #439: Some npm packages are outdated or are missing
**Actual Behaviour**
npm throws warnings about some packages

**Expected Behaviour**
npm warning should be taken care of.

**Steps to reproduce it**
Try npm install on your local deployment

**Screenshots of the issue**
w1

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by PrP-11**
Fixes #439 

Changes: Upgrade npm and fix npm warnings
**Note**: one warning cannot be fixed because it requires react 16 and upgrading to 16 will result of failure of other components

Surge Deployment Link: https://pr-441-fossasia-susi-accounts.surge.sh

Screenshots for the change:
Before:
w1
Now:
w6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "react-text-highlight": "0.2.0",
     "react-timezone": "1.0.5",
     "react-toolbox": "2.0.0-beta.13",
+    "react-transition-group": "^2.4.0",
```
Comment (akshatnitd) [inline package.json]: Remove `^`

File: package.json [hunk 2]
```diff
@@ -54,7 +55,7 @@
     "eslint-plugin-jsx": "0.0.2",
     "eslint-plugin-jsx-a11y": "4.0.0",
     "eslint-plugin-prettier": "2.6.2",
-    "eslint-plugin-react": "7.10.0",
+    "eslint-plugin-react": "^6.10.3",
```
Comment (akshatnitd) [inline package.json]: Remove the ^ at all places

File: package.json [hunk 3]
```diff
@@ -11,7 +11,6 @@
     "contrast": "1.0.1",
     "country-data": "0.0.31",
     "eslint-config-react-app": "0.6.2",
```
Comment (akshatnitd) [inline package.json]: Move it to devDependency

File: package.json [hunk 4]
```diff
@@ -24,6 +23,7 @@
     "material-ui-password-field": "1.4.3",
     "md5": "2.2.1",
     "moment": "2.22.2",
+    "npm": "^6.3.0",
```
[the remark this answers is not in the document]
Comment (anshumanv) [reply package.json]: 😆

[the remark this answers is not in the document]
Comment (PrP-11) [reply package.json]: @akshatnitd My npm version is already upgraded, this is to solve the upgrade warning thrown by travis as pointed out my mario on gitter

File: package.json [hunk 5]
```diff
@@ -24,6 +22,7 @@
     "material-ui-password-field": "1.4.3",
     "md5": "2.2.1",
     "moment": "2.22.2",
+    "npm": "6.3.0",
```
Comment (akshatnitd) [inline package.json]: Should should not be added. Make changes in travis config if you wish to upgrade npm.

File: package.json [hunk 6]
```diff
@@ -54,7 +54,8 @@
     "eslint-plugin-jsx": "0.0.2",
     "eslint-plugin-jsx-a11y": "4.0.0",
     "eslint-plugin-prettier": "2.6.2",
-    "eslint-plugin-react": "7.10.0",
+    "eslint-plugin-react": "6.10.3",
+    "eslint-config-react-app": "0.6.2",
```
Comment (akshatnitd) [inline package.json]: Follow alphabetical order

**Comment by akshatnitd** [comment]
@PrP-11 Please finalise the PR.

**Comment by PrP-11** [comment]
@akshatnitd made the changes. Removing the `"npm": "6.3.0"` is failing the travis build. Please review.
travis

**Pull request merged**